setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup, find_packages
+
```

Comment:
```
from setuptools import setup

requirements = [
    "cffi==1.15.0",
    "cryptography==37.0.4",
    "pycparser==2.21",
    "secp256k1==0.14.0",
    "websocket-client==1.3.3",
]

test_requirements = [
    'pytest',
]

with open("README.md", "r") as f:
    long_description = f.read()

setup(
    name='nostr',
    version="0.0.1",
    packages=find_packages(include=['nostr']),
    python_requires='>3.6.0',
    install_requires=requirements,
    test_requires=test_requirements,
    url='https://github.com/jeffthibault/python-nostr',
    description="A Python library for making Nostr clients.",
    long_description=long_description,
    long_description_content_type="text/markdown",
    classifiers=[
        'Operating System :: POSIX :: Linux',
        'Operating System :: Microsoft :: Windows',
        'Operating System :: MacOS :: MacOS X',
    ],
)
```

Reply:
I added the extra details, but for now I'm omitting the dependencies as those are already defined in the `requirements.txt` and would be a pain to maintain duplicate entries. When an actual pypi package is in the works, code can be added here to parse the `requirements.txt` and inject them into the `setup()` call.

I think this approach is more friendly to devs working on their local branches who may be tweaking `requirements.txt` deps.

Same for the `test_requirements`.